applications/electron/electron-builder.yml

```diff
@@ -48,6 +48,9 @@ mac:
 linux:
   icon: resources/icons/LinuxLauncherIcon/512-512.png
   category: Development
+  mimeTypes:
+    - text/plain
```

Comment:
I believe that by default, Theia assumes any file passed to it is a persisted (potentially multi-root) workspace. 
Therefore, adding `text/plain` may not achieve what a user might expect, which would be to open the file in an editor. Instead an empty workspace will be opened. 
Do you also see this?

Reply:
Yes, confirmed. Single file works in VS Code but not in Theia.